superset/assets/src/dashboard/components/gridComponents/Markdown.jsx

```diff
@@ -172,12 +186,14 @@ class Markdown extends React.PureComponent {
   }
 
   renderPreviewMode() {
-    return (
+    const markdownComponent = (
```

Comment:
Why do we need a `const` instead of directly return?

Reply:
fixed.